<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LPANDIRI-E7440" failures="1" tests="1" name="com.qa.e2e.CTMWorkflow.tests.New_TC06_CTM_VerifyClaimStatusAfterTreatmentInAdmin" time="52.096" errors="0" timestamp="2022-02-03T22:12:19 EST" skipped="0">
  <testcase classname="com.qa.e2e.CTMWorkflow.tests.New_TC06_CTM_VerifyClaimStatusAfterTreatmentInAdmin" name="VerifyClaimAndPaymentStatusBeforePaymentApprovalInAdminPortal" time="52.096">
    <failure message="expected [POS Pre-Authorization] but found [POS Purchase]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [POS Pre-Authorization] but found [POS Purchase]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.juno.qa.pages.E2EWorkflowPage.VerifyTransactionsTabDetails(E2EWorkflowPage.java:537)
at com.juno.qa.pages.E2EWorkflowPage.VerifyStatusAcrossPagesBeforePaymentApprovalInAdminPortal(E2EWorkflowPage.java:66)
at com.qa.e2e.CTMWorkflow.tests.New_TC06_CTM_VerifyClaimStatusAfterTreatmentInAdmin.VerifyClaimAndPaymentStatusBeforePaymentApprovalInAdminPortal(New_TC06_CTM_VerifyClaimStatusAfterTreatmentInAdmin.java:121)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- VerifyClaimAndPaymentStatusBeforePaymentApprovalInAdminPortal -->
  <system-out/>
</testsuite> <!-- com.qa.e2e.CTMWorkflow.tests.New_TC06_CTM_VerifyClaimStatusAfterTreatmentInAdmin -->
